build.gradle

```diff
@@ -636,6 +653,10 @@ String getVersionFromFile() {
   return file('version.txt').text.trim()
 }
 
+String gitBuildSha() {
```

Comment:
I don't think this is actually needed. We can just use `versionDetails().gitHash` from the `gradle-git-version` plugin